### ISSUE TITLE
Integration tests started to fail because of workmanager

### ISSUE THREAD

**Issue by vusters** [EnsembleUI/ensemble#566]
Integration tests started to fail due to:
```
[!] Unable to determine Swift version for the following pods:

    - `workmanager` does not specify a Swift version and none of the targets (`Runner`) integrating it have the `SWIFT_VERSION` attribute set. Please contact the author or set the `SWIFT_VERSION` attribute in at least one of the targets that integrate this pod.
```

**Comment by vusters** [comment]
@snehmehta putting this on your queue.

**Comment by vinothvino42** [comment]
Fixed by adding bridging header, PR - #585 
I tried running the test and these are the logs

```
ensemble git:(integration-test-issue) ✗ flutter test integration_test
Connected devices:
Vinoth’s iPhone (mobile) • ios • iOS 16.5 202
iPhone 14 (mobile)       • ios • com.apple.CoreSimulator.SimRuntime.iOS-16-4 (simulator)

No wireless devices were found.

[1]: Vinoth’s iPhone ()
[2]: iPhone 14 ()
Please choose one (or "q" to quit): 1
00:16 +0: loading /Users/vinothvino/Documents/Zyngl/EnsembleUI/ensemble/integration_test/themedApp_test.dart                                                                                                      Automatically signing iOS for device deployment using specified development team in Xcode project: HW2VJEFH2M
00:21 +0: loading /Users/vinothvino/Documents/Zyngl/EnsembleUI/ensemble/integration_test/themedApp_test.dart                                                                                                      Ru00:33 +0: loading /Users/vinothvino/Documents/Zyngl/EnsembleUI/ensemble/integration_test/themedApp_test.dart                                                                                                  11.1s
Upgrading Pods-Runner-frameworks.sh
02:37 +0: loading /Users/vinothvino/Documents/Zyngl/EnsembleUI/ensemble/integration_test/themedApp_test.dart                                                                                                       
02:43 +0: loading /Users/vinothvino/Documents/Zyngl/EnsembleUI/ensemble/integration_test/themedApp_test.dart                                                                                                   6.6s
Xcode build done.                                           130.8s
02:44 +0: loading /Users/vinothvino/Documents/Zyngl/EnsembleUI/ensemble/integration_test/themedApp_test.dart                                                                                                       I04:40 +0: loading /Users/vinothvino/Documents/Zyngl/EnsembleUI/ensemble/integration_test/themedApp_test.dart                                                                                                 115.9s
04:41 +0: /Users/vinothvino/Documents/Zyngl/EnsembleUI/ensemble/integration_test/themedApp_test.dart: Default App Tests Test Widgets inheriting from themes                                                        
══╡ EXCEPTION CAUGHT BY FLUTTER TEST FRAMEWORK ╞════════════════════════════════════════════════════
The following TestFailure was thrown running a test:
Expected: exactly one matching node in the widget tree
  Actual: _AncestorFinder:<zero widgets with type "TextField" which is an ancestor of text
"original">
   Which: means none were found but one was expected

When the exception was thrown, this was the stack:
#4      main.<anonymous closure>.<anonymous closure> (file:///Users/vinothvino/Documents/Zyngl/EnsembleUI/ensemble/integration_test/themedApp_test.dart:29:7)
<asynchronous suspension>
<asynchronous suspension>
(elided one frame from package:stack_trace)

This was caught by the test expectation on the following line:
  file:///Users/vinothvino/Documents/Zyngl/EnsembleUI/ensemble/integration_test/themedApp_test.dart line 29
The test description was:
  Test Widgets inheriting from themes
════════════════════════════════════════════════════════════════════════════════════════════════════
04:42 +0 -1: /Users/vinothvino/Documents/Zyngl/EnsembleUI/ensemble/integration_test/themedApp_test.dart: Default App Tests Test Widgets inheriting from themes [E]                                                 
  Test failed. See exception logs above.
  The test description was: Test Widgets inheriting from themes
  

To run this test again: /Users/vinothvino/Developer/flutter/bin/cache/dart-sdk/bin/dart test /Users/vinothvino/Documents/Zyngl/EnsembleUI/ensemble/integration_test/themedApp_test.dart -p vm --plain-name 'Default App Tests Test Widgets inheriting from themes'
05:02 +0 -1: loading /Users/vinothvino/Documents/Zyngl/EnsembleUI/ensemble/integration_test/defaultApp_test.dart                                                                                                  Automatically signing iOS for device deployment using specified development team in Xcode project: HW2VJEFH2M
05:06 +0 -1: loading /Users/vinothvino/Documents/Zyngl/EnsembleUI/ensemble/integration_test/defaultApp_test.dart                                                                                                  Ru05:39 +0 -1: loading /Users/vinothvino/Documents/Zyngl/EnsembleUI/ensemble/integration_test/defaultApp_test.dart                                                                                                   
05:43 +0 -1: loading /Users/vinothvino/Documents/Zyngl/EnsembleUI/ensemble/integration_test/defaultApp_test.dart                                                                                               4.5s
Xcode build done.                                           36.9s
05:44 +0 -1: loading /Users/vinothvino/Documents/Zyngl/EnsembleUI/ensemble/integration_test/defaultApp_test.dart                                                                                                   I07:07 +0 -1: loading /Users/vinothvino/Documents/Zyngl/EnsembleUI/ensemble/integration_test/defaultApp_test.dart                                                                                              83.1s
07:08 +0 -1: /Users/vinothvino/Documents/Zyngl/EnsembleUI/ensemble/integration_test/defaultApp_test.dart: Default App Tests Bindings to widget's value                                                             
══╡ EXCEPTION CAUGHT BY FLUTTER TEST FRAMEWORK ╞════════════════════════════════════════════════════
The following TestFailure was thrown running a test:
Expected: exactly one matching node in the widget tree
  Actual: _DescendantFinder:<zero widgets with text "first" that has ancestor(s) with type
"EnsembleText" (ignoring offstage widgets)>
   Which: means none were found but one was expected

When the exception was thrown, this was the stack:
#4      main.<anonymous closure>.<anonymous closure> (file:///Users/vinothvino/Documents/Zyngl/EnsembleUI/ensemble/integration_test/defaultApp_test.dart:29:7)
<asynchronous suspension>
<asynchronous suspension>
(elided one frame from package:stack_trace)

This was caught by the test expectation on the following line:
  file:///Users/vinothvino/Documents/Zyngl/EnsembleUI/ensemble/integration_test/defaultApp_test.dart line 29
The test description was:
  Bindings to widget's value
════════════════════════════════════════════════════════════════════════════════════════════════════
07:08 +0 -2: /Users/vinothvino/Documents/Zyngl/EnsembleUI/ensemble/integration_test/defaultApp_test.dart: Default App Tests Bindings to widget's value [E]                                                         
  Test failed. See exception logs above.
  The test description was: Bindings to widget's value
  

To run this test again: /Users/vinothvino/Developer/flutter/bin/cache/dart-sdk/bin/dart test /Users/vinothvino/Documents/Zyngl/EnsembleUI/ensemble/integration_test/defaultApp_test.dart -p vm --plain-name 'Default App Tests Bindings to widget'\''s value'
07:09 +0 -2: /Users/vinothvino/Documents/Zyngl/EnsembleUI/ensemble/integration_test/defaultApp_test.dart: Default App Tests API Binding                                                                            
══╡ EXCEPTION CAUGHT BY FLUTTER TEST FRAMEWORK ╞════════════════════════════════════════════════════
The following TestFailure was thrown running a test:
Expected: exactly one matching node in the widget tree
  Actual: _DescendantFinder:<zero widgets with text "count " that has ancestor(s) with type
"EnsembleText" (ignoring offstage widgets)>
   Which: means none were found but one was expected

When the exception was thrown, this was the stack:
#4      main.<anonymous closure>.<anonymous closure> (file:///Users/vinothvino/Documents/Zyngl/EnsembleUI/ensemble/integration_test/defaultApp_test.dart:76:7)
<asynchronous suspension>
<asynchronous suspension>
(elided one frame from package:stack_trace)

This was caught by the test expectation on the following line:
  file:///Users/vinothvino/Documents/Zyngl/EnsembleUI/ensemble/integration_test/defaultApp_test.dart line 76
The test description was:
  API Binding
════════════════════════════════════════════════════════════════════════════════════════════════════
07:09 +0 -3: /Users/vinothvino/Documents/Zyngl/EnsembleUI/ensemble/integration_test/defaultApp_test.dart: Default App Tests API Binding [E]                                                                        
  Test failed. See exception logs above.
  The test description was: API Binding
  

To run this test again: /Users/vinothvino/Developer/flutter/bin/cache/dart-sdk/bin/dart test /Users/vinothvino/Documents/Zyngl/EnsembleUI/ensemble/integration_test/defaultApp_test.dart -p vm --plain-name 'Default App Tests API Binding'
07:09 +0 -3: Some tests failed.                                                                                                         
```

**Comment by vusters** [comment]
Awesome @vinothvino42 . Please use this jira for check in.
